buildSrc/src/main/java/com/osacky/flank/gradle/YamlWriter.kt

```diff
@@ -57,7 +62,7 @@ internal class YamlWriter {
     appendProperty(config.outputStyle, name = "output-style")
   }
 
-  internal fun writeAdditionalProperties(config: FladleConfig): String = buildString {
+  internal fun writeAdditionalProperties(config: FladleConfig, printRobo: Boolean = true): String = buildString {
```

Comment:
instead of having a boolean passed in to this function, maybe we can just separate it in to two functions? I think that would make the code more readable.
i have an intrinsic dislike of boolean parameters to functions 😆

Reply:
Actually, I can remove this logic entirely since now we have `sanityRobo` in `FladleConfig`